src/python/tests/core/bot/testcase_manager_test.py

```diff
@@ -758,6 +760,7 @@ def setUp(self):
     """Set up."""
     super(UntrustedEngineReproduceTest, self).setUp()
     environment.set_value('JOB_NAME', 'libfuzzer_asan_job')
+    environment.set_value('USE_MINIJAIL', False)
```

Comment:
@oliverchang - there is some carryover from corpus pruning test here. if i remove this, tests fail on CI weirdly, even though locally they succeed, if you can please help to make a quick check, that would be appreciated.